script/preprocessing.py

```diff
@@ -111,11 +116,94 @@ def musdb(file_path):
             else:
                 sound_class = None
 
-        else:
-            instrument_name = None
-            sound_class = None
 
-    return tempo, instrument_name, key, sound_class            
+    return tempo, instrument_name, key, sound_class
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+            prog="PreprocessingHelper",
+            description="This script creates metadata for BRID and/or MUSDB"
+            )
+
+    # arguments. required --> data home, dataset (if using preprocessing)
+    parser.add_argument("--data_home", required=True, help="pathway to where is data is stored")
+    parser.add_argument("--dataset", required=True, help="supported datasets: BRID (enter 'brid') and MUSDB (enter 'musdb')")
+    parser.add_argument("--track_files", help="txt file with track names")
+    # need to develop this still
+
+    parser.add_argument("--sr", required=False, default=44100, help="sample rate, default is 44100Hz")
+    parser.add_argument("--duration", required=False, default=10.0, help="mixture duration, default is 10 seconds")
+    parser.add_argument("--n_mixtures", required=False, default=5, help="number of mixtures created")
+    parser.add_argument("--n_stems", required=False, default=3, help="number of stems pertaining to each mix")
+    parser.add_argument("--n_harmonic", required=False, default=0, help="number of harmonic stems")
+    parser.add_argument("--n_percussive", required=False, default=0, help="number of percussive stems")
+
+
+    args = parser.parse_args()
+    kwargs = vars(args)
+    path_to_stems = os.path.join(args.data_home, "stems")
```

Comment:
you can remove this since the data_home is already referring to the folder where the stems should be